Tests/LibGfx/TestColor.cpp

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2023, Nico Weber <thakis@chromium.org>
```

Comment:
🤨

Reply:
I might or might not have copied TestDeltaE.cpp to create this file :eyes: